### PR TITLE
fix: improve resolver test suite

### PR DESCRIPTION
**Pull request by rjwalters**
Add tests for edge cases in scale

### CODE REVIEW

File: mettagrid/mettagrid/resolvers.py [hunk 1]
```diff
@@ -109,6 +109,17 @@ def oc_scale(
     Numeric
         The scaled value in the output range
     """
+
+    # Ensure in_min <= in_max
+    assert in_min <= in_max
+    assert out_min <= out_max
+
+    if in_min == in_max:
+        return (out_min + out_max) / 2
```
Comment (sasmith) [inline mettagrid/mettagrid/resolvers.py:118]: This seems wrong, or at least dicey. Can we just assert `in_min < in_max`?

**Review by sasmith — APPROVED** [review]

File: mettagrid/mettagrid/resolvers.py (hunk 1 above)
Comment (rjwalters) [reply mettagrid/mettagrid/resolvers.py]: yeah, we can define it that way instead
